src/templates/ChaftaCoo/ChaftaCooTemplate.tsx

```diff
@@ -354,8 +364,9 @@ export const CertificationSection: FunctionComponent<TemplateProps<ChaftaCooDocu
 export const ChaftaCooTemplate: FunctionComponent<TemplateProps<ChaftaCooDocument> & HasPrivacyToggle> = (props) => {
   const [isPrivacyOn, setIsPrivacyOn] = useState(false);
```

Comment:
want to pass in the specific props into the sections at this level? rather then repeating the get `documentData`

Reply:
ah okay, done.